### PR TITLE
[12.0][ADD] account_bank_receipt

### PR DESCRIPTION
**Pull request by Saran440**
Base on account_check_deposit

### CODE REVIEW

**Comment by kittiu** [comment]
@Saran440 please fix the travis.

File: account_bank_receipt/data/sequence.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
```
Comment (kittiu) [inline account_bank_receipt/data/sequence.xml:2]: I think you don't need to copy all the credit from all xml files.
However, you can add comment in Description section of readme, that this module is based on the work of account_check_deposit, but make it less specific to cover more use case.

**Comment by kittiu** [comment]
Please start from your fork's new branch, i..e, 12.0-add-account_bank_receipt

File: account_bank_receipt/report/report.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  Copyright (C) 2014-2019 Akretion (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  The licence is in the file __manifest__.py
+-->
+
+<odoo>
+
+    <report id="report_account_bank_receipt"
+            model="account.bank.receipt"
+            string="Check Deposit"
```
Comment (kittiu) [inline account_bank_receipt/report/report.xml:13]:
```suggestion
            string="Bank Receipt"
```
**Note** I think wording check deposit should change to "bank receipt" on all places.

**Comment by Saran440** [comment]
change branch to 12.0-add-account_bank_receipt

**Pull request closed without merging**